modules/serverless/pages/serving/serving-transport-encryption.adoc

```diff
@@ -0,0 +1,446 @@
+
+==== External domain encryption
+
+The transport encryption for external domains is handled by the cluster's ingress layer.
+This is either {product-title} ingress or {smproductname}.
+Please refer to the relevant documentation for more information.
+
+==== Cluster-local encryption
+
+image::serving-encryption/encryption-cluster-local-domain.drawio.svg[]
+
+This part enables transport encryption for cluster-local domains. It has the following properties:
+
+* Certificate CN/SAN contains the cluster-local domains of a Knative Service, e.g. `myapp.namespace.svc.cluster.local`, `myapp.namespace.svc`, `myapp.namespace`.
+* The certificates are hosted using SNI by the cluster-local endpoint of the ingress-controller.
+* The caller has to trust the CA that signed the certificates (this is out of the scope of {ServerlessProductName}).
+* To create the certificates, Knative relies on cert-manager. cert-manager needs to be installed and configured for the feature to work.
+
+==== {ServerlessProductName} Serving system-internal encryption
+
+image::serving-encryption/encryption-system-internal.drawio.svg[]
+
+This part enables transport encryption for system internal components (Ingress-Gateway, Activator, Queue-Proxy). These components are hosting TLS endpoints when this configuration is used:
+
+* To get the certificates, {ServerlessProductName} relies on cert-manager. cert-manager needs to be installed and configured for the feature to work.
+* Specific SANs are used to verify each connection. Each component needs to trust the CA that signed the certificates. For this, {ServerlessProductName} system components will consume and trust a bundle of CAs. The CA bundle needs to be provided by the cluster administrator.
+
+
+== Identify and select an appropriate certificate issuer [[issuer_selection]]
+
+[INFO]
+====
+Issuers here are referring to cert-manager `Issuers` and `ClusterIssuers`. They represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests.
+Reference: https://cert-manager.io/docs/concepts/issuer/
+====
+
+In general, you can refer to the cert-manager documentation (https://cert-manager.io/docs/configuration/issuers/). There are examples available for:
+
+* A custom CA stored in a K8s secret
+* HTTP-01 challenges, e.g. Let's encrypt
+* DNS-01 challenges
+* Self-signed issuers
+
+[IMPORTANT]
+====
+Please note, that not all issuer types work for each Knative Serving encryption feature. See below for details.
+====
+
+Depending on the feature you would like to use, {serverlessproductname} requires your certificate issuer to be capable of signing the following certificates:
+
+For the `cluster-local-domain-tls` feature, the issuer needs to be able to sign certificates for cluster-local domains like:
```

Comment:
You should settle on one wording, "part" or "feature". Also `cluster-local-domain-tls` was referred to as "Cluster-local encryption" above, you probably should stick to the same wording. Maybe you just write here "For cluster-local encryption, the issuer ..."